generate-docs/GenerateDocs.sh

```diff
@@ -1,3 +1,4 @@
+#!/bins/sh
```

Comment:
Shouldn't this be `bin`, not `bins`? Granted, I only ever run this on Windows and haven't seen the error in question myself...

Reply:
You're absolutely right, no idea how that slipped by me.

The error is given by the [ShellCheck](https://marketplace.visualstudio.com/items?itemName=timonwong.shellcheck) extension, it needs a shebang to know what to check for :)